package.json

```diff
@@ -15,13 +15,17 @@
     "@mui/icons-material": "^5.14.3",
     "@mui/material": "^5.14.4",
     "@types/node": "20.5.0",
+    "@types/pdf-parse": "^1.1.1",
     "@types/react": "18.2.20",
     "@types/react-dom": "18.2.7",
     "autoprefixer": "10.4.15",
     "eslint": "8.47.0",
     "eslint-config-next": "13.4.13",
     "next": "13.4.13",
     "next-auth": "^4.23.0",
+    "node-fetch": "^3.3.2",
+    "npm": "^9.8.1",
```

Comment:
do we need these new packages? feel like they were added when you were trying out stuff. npm uninstall them if not needed

Reply:
I think we need node-fetch for the OpenAI API to work, getting rid of the other ones now.